src/main/resources/db/changeLog.xml

```diff
@@ -48,4 +48,10 @@
             <column name="isVacation" type="boolean"/>
         </addColumn>
     </changeSet>
+    <changeSet id="seperate_project_times" author="Ville Juutila">
+        <renameColumn tableName="TimeEntry" oldColumnName="projectTime" newColumnName="billableProjectTime" columnDataType="integer"/>
```

Comment:
It is not handled... I don't think it matters at this point when the application is not in production use. Also can't think of a way to handle it, since project billability has not been stored anywhere before this feature.